src/SFA.DAS.AssessorService.ExternalApiDataSync/Helpers/ManagedIdentitySqlConnection.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Azure.Services.AppAuthentication;
```

Comment:
This duplicates src/SFA.DAS.AssessorService.Data/Helpers/ManagedIdentitySqlConnection.cs.  Can the duplication be removed by aligning the target frameworks so you can reference the helper class in SFA.DAS.AssessorService.Data rather than duplicating code?

Reply:
The ExteranApiDataSync has been moved to the assessor functions repo so is not part of this PR.